xml/ha_storage_protection.xml

```diff
@@ -779,27 +782,67 @@ Timeout (msgwait)  : 10
     node needs to be fenced. </para>
   </sect1>
 
-  <sect1 xml:id="sec.ha..storageprotection.sgpersist">
+<!--  <sect1 xml:id="sec.ha.storage.protect.fencing">
+  <title>Storage-based Fencing</title>
+  <para>
+   You can reliably avoid split brain scenarios by using one or more
+   &stonith; Block Devices (SBD), <literal>watchdog</literal> support and
+   the <literal>external/sbd</literal> &stonith; agent.
+  </para>
+ </sect1>-->
+
+ <sect1 xml:id="sec.ha.storage.protect.morestuff">
+  <title>Additional Mechanisms for Storage Protection</title>
+  <remark>toms 2018-04-20: this can be improved...</remark>
+  <para>Apart from &stonith;, there are other methods to reach storage
+  protection. SCSI-3 and SCSI-4 uses persistent reservations whereas
+  <literal>sfex</literal> provides a locking mechanism. Both methods are
+   explained in the following subsections.
+  </para>
+  <sect2 xml:id="sec.ha.storage.protect.sgpersist">
    <title>Configuring an sg_persist Resource</title>
-   <remark>toms 2018-04-18: Not sure why this section is here. It does not
-    belong to the described procedure introduced in
-    <xref linkend="sec.ha.storageprotection.fencing.setup"/>.
-    I'm tempted to move it...</remark>
+   <remark>toms 2018-04-20: I would like to see a little bit more background
+   information. </remark>
+   <para>
+    The SCSI specification 3 and 4 defines <emphasis>persistent reservations</emphasis>.
+    These are SCSI protocol features and can be used for I/O fencing and failover.
+    This feature is implemented in the <command>sg_persist</command> Linux
+    command.
+   </para>
    <note>
     <title>SCSI Disk Compatibility</title>
     <para> Any backing disks for <literal>sg_persist</literal> must be SCSI
      disk compatible. <literal>sg_persist</literal> only works for devices like
-     SCSI disks or iSCSI LUNs. Do <emphasis>not</emphasis> use it for IDE,
-     SATA, or any block devices which do not support the SCSI protocol. </para>
+     SCSI disks or iSCSI LUNs.
+     <remark>toms 2018-04-20: What about FCoE, FC, iSER, SRP, Serial Attached SCSI (SAR)?</remark>
+     Do <emphasis>not</emphasis> use it for IDE, SATA, or any block devices
+     which do not support the SCSI protocol. </para>
    </note>
-   <remark>toms 2014-09-10: FATE#312345</remark>
+   <para>Before you proceed, first check if your disk supports
+    persistent reservations. Use the following command (replace
+     <replaceable>DISK</replaceable> with your device name):</para>
+    <screen>&prompt.root;<command>sg_persist</command> -n --in --read-reservation -d /dev/<replaceable>DISK</replaceable></screen>
+   <para>The result will show you if your disk is supports persistent
```

Comment:
typo (and you can use present tense for results, to keep it simple): The result shows if your disk supports persistent reservations